gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -160,6 +160,16 @@ typedef struct _GstTensorFilterProperties
   int throughput; /**< The average throughput in the number of outputs per second */
 } GstTensorFilterProperties;
 
+/**
+ * @brief Structure definition for tensor-filter framework statistics
+ */
+typedef struct _GstTensorFilterFrameworkStatistics
+{
+  int64_t total_invoke_num;      /**< The total number of calls to invoke */
+  int64_t total_invoke_latency;  /**< The total accumulated invoke latency (usec) */
+  int64_t total_overhead_latency;    /**< The total accumulated overhead latency from the extension (usec) */
+} GstTensorFilterFrameworkStatistics;
```

Comment:
I think this feature would be overlapped with `latency` and `throughput` in `GstTensorFilterProperties`.
Could you merge them into a unified one?

Reply:
The latency used by GstTensorFilterProperties measures the latency inside the tensor filter (measuring time around the invoke call). This one allows extensions to measures the latency themselves and tensor filter can extract it if needed.

I can merge the data structure into one. Is that what you are suggesting?

Reply:
Do not merge C-API's infra and NNStreamer-core's infra.